Sources/DangerShellExecutor/ShellExecutor.swift

```diff
@@ -19,14 +47,14 @@ internal class ShellExecutor {
         task.waitUntilExit()
 
         let data = pipe.fileHandleForReading.readDataToEndOfFile()
-        return String(data: data, encoding: String.Encoding.utf8)!
+        return String(data: data, encoding: String.Encoding.utf8)!.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerShellExecutor/ShellExecutor.swift;
  Line: 50;
-->

- :warning: Line should be 120 characters or less: currently 126 characters (`line_length`)